docs/sources/features/datasources/stackdriver.md

```diff
@@ -323,3 +323,12 @@ datasources:
     jsonData:
       authenticationType: gce
 ```
+
+## Deep linking from Grafana panels to the Metrics Explorer in Gooogle Monitoring
```

Comment:
Not sure if cloud should be included? 
```suggestion
## Deep linking from Grafana panels to the Metrics Explorer in Gooogle Cloud Monitoring
```

Reply:
I'm not completely sure about this; in the google console is referred only as Monitoring.

Reply:
That's fine. Probably me not understanding that you're referring to the console with "Gooogle Monitoring".